test/tools/ossfuzz/abiV2FuzzerCommon.cpp

```diff
@@ -0,0 +1,32 @@
+#include <test/tools/ossfuzz/abiV2FuzzerCommon.h>
+
+using namespace dev::test::abiv2fuzzer;
+
+SolidityCompilationFramework::SolidityCompilationFramework(langutil::EVMVersion _evmVersion)
+{
+	m_evmVersion = _evmVersion;
+}
+
+dev::bytes SolidityCompilationFramework::compileContract(std::string const& _sourceCode)
+{
+	// Silence compiler version warning
+	std::string sourceCode = "pragma solidity >=0.0;\n";
+	sourceCode += "pragma experimental ABIEncoderV2;\n";
+	sourceCode += _sourceCode;
+	m_compiler.setSources({{"", sourceCode}});
+	m_compiler.setEVMVersion(m_evmVersion);
+	m_compiler.setOptimiserSettings(m_optimiserSettings);
+	if (!m_compiler.compile())
+	{
+		langutil::SourceReferenceFormatter formatter(std::cerr);
+
+		for (auto const& error: m_compiler.errors())
+			formatter.printExceptionInformation(
+					*error,
+					formatter.formatErrorInformation(*error)
+			);
+		std::cerr << "Compiling contract failed" << std::endl;
```

Comment:
This will still access the bytecode - shouldn't it fail in some way?

Reply:
Right! Will fix this soon.

Reply:
I fixed this by throwing an assertion that is meant to be caught by the API caller.